### PR TITLE
FMC doc cleanup and PCR update

### PR DESCRIPTION
**Pull request by FerralCoder**
Cleaned up some outstanding TODO items in FMC doc and added more detail for PCR flows.

### CODE REVIEW

**Comment by FerralCoder** [comment]
In creating this PR, I noticed some problems and inconsistencies in the PCR handling from ROM and FMC. Documented in this issue:

https://github.com/chipsalliance/caliptra-sw/issues/449

**Review by rusty1968 — APPROVED** [review]

File: fmc/README.md [hunk 1]
```diff
@@ -11,7 +11,10 @@
 |            |         | Add details/clarifications in FMC Boot steps including where to store artifacts    |
 |            |         | Add Firmware Handoff Table (FHT) definition                                        |
 | 03/21/2023 | 0.5     | Additional fields added to FHT                                                     |
-| 03/29/2023 | 0.5     | Changed the value for invalid adress fields in FHT                                                     |
+| 03/29/2023 | 0.5.1   | Changed the value for invalid adress fields in FHT                                 |
```
Comment (mhatrevi) [inline fmc/README.md:14]: NIT: address

File: fmc/README.md [hunk 2]
```diff
@@ -211,6 +214,13 @@ This field provides the Handle into the Data Vault where the SVN<sub>RT</sub> is
 
 This area is reserved for definition of additional fields that may be added during Minor version updates of the FHT.
 
+## PCR Registers
+
+FMC has the responsibility to update 2 PCR registers.<br>
+FMC updates PCR2 to reflect the firmware update Journey with measurements of RT firmware and FW Manifest. This register is only cleared on cold reset.<br>
+FMC updates PCR2 to reflect only the Current running firmware with measurements of RT firmware and FW Manifest. This register is cleared on all reset types.<br>
+FMC locks its PCR registers before handing control to RT firmware so that they may not me modified later in the boot.
```
Comment (mhatrevi) [inline fmc/README.md:222]: be*

**Review by mhatrevi — APPROVED** [review]

File: fmc/README.md (hunk 1 above)
Comment (FerralCoder) [reply fmc/README.md]: fixed

File: fmc/README.md (hunk 2 above)
Comment (FerralCoder) [reply fmc/README.md]: fixed

**Review by rusty1968 — APPROVED** [review]

**Pull request merged**